index.html

```diff
@@ -466,347 +466,365 @@ <h3>Ethereum EIP712 Signature 2021</h3>
                 },
                 {
                   "name": "name",
-                  "type": "string"
-                },
-                {
-                  "name": "child",
-                  "type": "Person"
+                  "type": "Name"
                 }
               ],
-              "EIP712Domain": [
+              "Document": [
                 {
-                  "name": "name",
-                  "type": "string"
+                  "name": "@context",
+                  "type": "string[]"
                 },
                 {
-                  "name": "version",
+                  "name": "@type",
                   "type": "string"
                 },
                 {
-                  "name": "chainId",
-                  "type": "uint256"
+                  "name": "data",
+                  "type": "Data"
                 },
                 {
-                  "name": "salt",
-                  "type": "bytes32"
-                }
-              ],
-              "Person": [
-                {
-                  "name": "type",
-                  "type": "string"
-                },
-                {
-                  "name": "name",
+                  "name": "telephone",
                   "type": "string"
                 }
               ],
-              "Proof": [
-                {
-                  "name": "verificationMethod",
-                  "type": "string"
-                },
+              "Job": [
                 {
-                  "name": "created",
+                  "name": "employer",
                   "type": "string"
                 },
                 {
-                  "name": "proofPurpose",
-                  "type": "string"
-                },
-                {
-                  "name": "type",
+                  "name": "jobTitle",
                   "type": "string"
                 }
               ],
-              "VerifiableCredential": [
-                {
-                  "name": "@context",
-                  "type": "string[]"
-                },
-                {
-                  "name": "type",
-                  "type": "string[]"
-                },
-                {
-                  "name": "id",
-                  "type": "string"
-                },
+              "Name": [
                 {
-                  "name": "issuer",
+                  "name": "firstName",
                   "type": "string"
                 },
                 {
-                  "name": "issuanceDate",
+                  "name": "lastName",
                   "type": "string"
-                },
-                {
-                  "name": "credentialSubject",
-                  "type": "CredentialSubject"
-                },
-                {
-                  "name": "credentialSchema",
-                  "type": "CredentialSchema"
-                },
-                {
-                  "name": "proof",
-                  "type": "Proof"
                 }
```

Comment:
The `proof` property and `Proof` type is missing. Understandably, the signature (`proof.proofValue`) cannot be included in the signing input, nor can practically the embedded types (`proof.eip712Domain`), but shouldn't the other proof properties (`created`, `proofPurpose`, `type`, `verificationMethod`) be included in the data to be signed?

Reply:
Hm interesting. My understanding was we don't sign over the data in the `proof` object and use it more as a way to provide options and configurability. Will work on fixing this.

Reply:
I see it's fixed in [28bd5ed](https://github.com/w3c-ccg/ethereum-eip712-signature-2021-spec/pull/26/commits/28bd5edecde8395242aea8ba64e9be25f59585d0). Thanks!